packages/primereact/package.json

```
@@ -0,0 +1,97 @@
+{
+  "name": "@rjsf/primereact",
+  "version": "6.0.0-alpha.0",
+  "description": "PrimeReact theme, fields and widgets for react-jsonschema-form",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
```

@jasny I improved ESM support in the other themes in `5.24.7`. Can you add these:
```suggestion
  "typings": "lib/index.d.ts",
  "exports": {
    ".": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./lib/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    },
    "./dist": {
      "require": "./dist/index.js",
      "import": "./lib/index.js",
      "types": "./lib/index.d.ts"
    },
    "./dist/*.js": {
      "require": "./dist/*.js",
      "import": "./lib/*.js",
      "types": "./lib/*.d.ts"
    }
  },
  "scripts": {
```